### PR TITLE
ddl: update ddl doc

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 

Update doc and add faq for ddl.

### What is the related PR or file link(s)? 

No PR.

### Which version does your change affect? 

V2.1, V3.0

### CODE REVIEW

File: v3.0/faq/tidb.md [hunk 1]
```diff
@@ -555,6 +555,11 @@ TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specif
 同 MySQL 的用法一致，例如：
 `select column_name from table_name use index（index_name）where where_condition;`
 
+#### 3.3.13 高并发执行 DDL 报错？
+
+高并发情况下执行 DDL （比如批量建表）时，极少部分 DDL 可能会执行失败。原因是并发执行 DDL 时存在冲突 key，并发太高导致冲突很严重时，可能会导致 DDL 执行失败。
+建议 DDL 并发不要超过20。否则应用端需要自己处理 DDL 失败然后重试。
```
Comment (crazycs520) [inline v3.0/faq/tidb.md]: "并发不要超过20", 是通过本地测试得出的结论，并发高到 50的话跑一会儿就会有一个 冲突 error.

Comment (lonng) [inline v3.0/faq/tidb.md]:
```suggestion
建议 DDL 并发低于20。否则需要在应用端重试失败的 DDL 语句。
```

File: v3.0/faq/tidb.md [hunk 2]
```diff
@@ -555,6 +555,11 @@ TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specif
 同 MySQL 的用法一致，例如：
 `select column_name from table_name use index（index_name）where where_condition;`
 
+#### 3.3.13 高并发执行 DDL 报错？
+
+高并发情况下执行 DDL （比如批量建表）时，极少部分 DDL 可能会执行失败。原因是并发执行 DDL 时存在冲突 key，并发太高导致冲突很严重时，可能会导致 DDL 执行失败。
```
Comment (lonng) [inline v3.0/faq/tidb.md]:
```suggestion
高并发情况下执行 DDL （比如批量建表）时，极少部分 DDL 可能会由于并发执行时 key 冲突而执行失败。
```

**Comment by lonng** [comment]
LGTM

**Comment by winkyao** [comment]
@lilin90 PTAL

**Pull request merged**